census/handler.go

```diff
@@ -335,7 +369,11 @@ func (m *Manager) Handler(ctx context.Context, r *api.MetaRequest, isAuth bool,
 		}
 		resp.URI = m.RemoteStorage.URIprefix() + cid
 		log.Infof("published census at %s", resp.URI)
-		resp.Root = tr.Root()
+		resp.Root, err = tr.Root()
+		if err != nil {
+			resp.SetError(err.Error())
+			return resp
+		}
```

Comment:
```suggestion
		resp.Root = root
```
I think you can reuse the variable from before